<?xml version="1.0"?>
<launch>
    <arg name="debug_plot" default="False"/>

    <!--  launch velocity scaler -->
    <node pkg="sector_tuner" type="vel_scaler_node.py" name="velocity_scaler" output="screen">
        <param name="debug_plot" value="$(arg debug_plot)"/>
    </node>
    
</launch>

